_data/versions.yml

```diff
@@ -1,3 +1,103 @@
+v3.1:
+- title: v3.1.0-rc1
+  note: |
+  18 March 2018
+
+  #### Support for OpenStack
+  Calico v3.1 reintroduces support for OpenStack. Users can now upgrade their Calico OpenStack clusters to v3.1 to take advantage of the new capabilities
+  introduced in the Calico v3 release series. To upgrade an existing cluster, [follow the documented procedure](https://docs.projectcalico.org/v3.1/getting-started/openstack/upgrade/).
```

Comment:
To the extent you could break the lines around 70-80 characters it will become easier to read in github.com.